<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>561</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>categories</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>upload</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(61,80,255);
font 75 10pt &quot;MS Shell Dlg 2&quot;;
font: 9pt &quot;MV Boli&quot;;</string>
     </property>
     <property name="text">
      <string>image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(170, 255, 255);
color:black
}</string>
     </property>
     <property name="text">
      <string>satistique</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(61,80,255);
font 75 10pt &quot;MS Shell Dlg 2&quot;;
font: 9pt &quot;MV Boli&quot;;</string>
     </property>
     <property name="text">
      <string>livraison</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_34">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(170, 255, 255);
color:black
}
</string>
     </property>
     <property name="text">
      <string>excel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(61,80,255);
font 75 10pt &quot;MS Shell Dlg 2&quot;;
font: 9pt &quot;MV Boli&quot;;</string>
     </property>
     <property name="text">
      <string>id categorie</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1 solid white;
border-radius: 5px;
background-color:rgba(0,0,0,15%);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(170, 255, 255);
color:black
}</string>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Downloads/Nouveau dossier/modifier.png</normaloff>../Downloads/Nouveau dossier/modifier.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(170, 255, 255);
color:black
}</string>
     </property>
     <property name="text">
      <string>suprimer</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Downloads/Nouveau dossier/supprimer.png</normaloff>../Downloads/Nouveau dossier/supprimer.png</iconset>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_17">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>201</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView {
   background-color:rgba(0,0,0,50%);

    }

</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(61,80,255);
font 75 10pt &quot;MS Shell Dlg 2&quot;;
font: 9pt &quot;MV Boli&quot;;</string>
     </property>
     <property name="text">
      <string>description</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>73</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>livrable</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>non livrable</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_17">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1 solid white;
border-radius: 5px;
background-color:rgba(0,0,0,15%);
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1 solid white;
border-radius: 5px;
background-color:rgba(0,0,0,15%);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(170, 255, 255);
color:black
}</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Downloads/Nouveau dossier/ajouter.png</normaloff>../Downloads/Nouveau dossier/ajouter.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>410</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(170, 255, 255);
color:black
}</string>
     </property>
     <property name="text">
      <string>translate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(61,80,255);
font 75 10pt &quot;MS Shell Dlg 2&quot;;
font: 9pt &quot;MV Boli&quot;;</string>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>831</width>
       <height>651</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Downloads/teeest.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>601</width>
       <height>461</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/teeest.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label</zorder>
    <zorder>pushButton</zorder>
    <zorder>label_28</zorder>
    <zorder>pushButton_7</zorder>
    <zorder>label_26</zorder>
    <zorder>pushButton_34</zorder>
    <zorder>label_20</zorder>
    <zorder>textEdit</zorder>
    <zorder>pushButton_13</zorder>
    <zorder>pushButton_17</zorder>
    <zorder>tableView_17</zorder>
    <zorder>label_25</zorder>
    <zorder>comboBox</zorder>
    <zorder>lineEdit_17</zorder>
    <zorder>lineEdit_13</zorder>
    <zorder>pushButton_14</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>label_19</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
